OpenRA.Mods.Common/Traits/Render/RenderSprites.cs

```diff
@@ -19,6 +19,12 @@
 
 namespace OpenRA.Mods.Common.Traits.Render
 {
+	public sealed class PlayerPaletteOverrideInit : ValueActorInit<string>, ISingleInstanceInit
+	{
+		public PlayerPaletteOverrideInit(string name)
+			: base(name) { }
+	}
+
```

Comment:
Shouldn't this be using the same palette overwriting as asset browser?

Reply:
Also IMO this should be using campaign colors / palettes. It feels wrong to have a color selection here

Reply:
Neither the asset browser (which only acts on a single sprite) or the color picker (which relies on a custom actor definition) approaches work here. I added the color picker because the neutral player colour looked bad - but I like the idea using the campaign colours based on selected style/faction.